packages/desktop/yarn.lock

```diff
@@ -1199,47 +1199,6 @@
     ajv "^6.12.0"
```

Comment:
Avoid committing `package.json` and associated lock files if you didn't actually modified anything in those regards - i'd suggest discarding this change altogether here - no reason to change these files in regards to this PR